iree/samples/static_library/README.md

```diff
@@ -41,8 +41,8 @@ for general instructions on building using CMake):
   ```shell
   cmake -B ../iree-build/
     -DIREE_BUILD_SAMPLES=ON \
-    -DIREE_TARGET_BACKENDS_TO_BUILD=DYLIB-LLVM-AOT \
-    -DIREE_HAL_DRIVERS_TO_BUILD=Dylib_Sync \
+    -DIREE_TARGET_BACKEND_DYLIB-LLVM-AOT=ON \
+    -DIREE_HAL_DRIVER_DYLIB_SYNC=ON \
```

Comment:
I think this is different. You need to also disable every other thing

Reply:
This doesn't _need_ the others disabled, but it does need _at least_ this compiler target backend and runtime HAL driver to be enabled.

Reply:
I meant "need" to not change the behavior. But if it doesn't need all the other backends, then building them seems like kind of a waste :-)

Reply:
Fair. Disabled the others by default in this documentation.